editor/store/selectors.js

```diff
@@ -1744,3 +1744,21 @@ export function getSupportedBlocks( state, uid, globallyEnabledBlockTypes ) {
 export function getEditorSettings( state ) {
 	return state.settings;
 }
+
+/*
+ * Returns the editor settings.
+ *
+ * @param {Object} state Editor state.
+ *
+ * @return {Object} The editor settings object
```

Comment:
This appears to return the locking setting, which is a string (with particular values). Looks like this needs updating.